### PR TITLE
Add coronavirus flag to investigations table

### PR DESCRIPTION
**Pull request by slorek**
https://trello.com/c/b3K7oCvN/466-add-page-asking-if-case-is-related-to-coronavirus-for-trading-standards

## Description
Adds the column required to record whether an investigation is related to the 2020 coronavirus pandemic. This is required for several other stories so I have separated it from the above linked story to unblock work on those.

Also contains schema updates missing from https://github.com/UKGovernmentBEIS/beis-opss-psd/pull/444

## Checklist:
- [x] Have you documented your changes in the pull request description?

### CODE REVIEW

File: psd-web/db/migrate/20200402142659_add_is_coronavirus_related_to_investigations.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddIsCoronavirusRelatedToInvestigations < ActiveRecord::Migration[5.2]
+  def change
+    safety_assured do
+      add_column :investigations, :is_coronavirus_related, :boolean, default: false
```
Comment (StupidCodeFactory) [inline psd-web/db/migrate/20200402142659_add_is_coronavirus_related_to_investigations.rb:4]: NITPICK:  I do like following the ruby guidelines on method naming https://rubystyle.guide/#bool-methods-prefix
```suggestion
      add_column :investigations, :coronavirus_related, :boolean, default: false
```

File: psd-web/db/schema.rb [hunk 2]
```diff
@@ -161,6 +161,7 @@
     t.text "hazard_description"
     t.string "hazard_type"
     t.boolean "is_closed", default: false
+    t.boolean "is_coronavirus_related", default: false
```
Comment (frankieroberto) [inline psd-web/db/schema.rb:164]: Please change: This no longer matches the migration... 😄

File: psd-web/db/migrate/20200402142659_add_is_coronavirus_related_to_investigations.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class AddIsCoronavirusRelatedToInvestigations < ActiveRecord::Migration[5.2]
+  def change
+    safety_assured do
+      add_column :investigations, :coronavirus_related, :boolean, default: false
+    end
```
Comment (frankieroberto) [inline psd-web/db/migrate/20200402142659_add_is_coronavirus_related_to_investigations.rb:5]: We _could_ avoid locking the whole table here (to add the default value), using one of the strategies [mentioned here](https://github.com/ankane/strong_migrations#adding-a-column-with-a-default-value) but it’s probably overkill at our scale...

Comment (slorek) [reply psd-web/db/migrate/20200402142659_add_is_coronavirus_related_to_investigations.rb]: Interestingly this raises a `Rails/ReversibleMigration` rubocop failure so I think I'll keep it as was.

Comment (frankieroberto) [reply psd-web/db/migrate/20200402142659_add_is_coronavirus_related_to_investigations.rb]: 👍

**Pull request merged**